src/gui/application.cpp

```diff
@@ -408,8 +408,7 @@ namespace Toolbox {
         if (FileDialog::Instance()->isDone()) {
             FileDialog::Instance()->Close();
             if (FileDialog::Instance()->isOk()) {
-                std::filesystem::path selected_path =
-                    FileDialog::Instance()->GetFilenameResult();
+                std::filesystem::path selected_path = FileDialog::Instance()->GetFilenameResult();
```

Comment:
What's your policy on breaking statements into multiple lines? I usually go with a roughly 80-100 character max width and break things logically so they don't go beyond there, because 80 is pretty standard and I like to have two panes side by side a lot when I code. Are you going with "never break lines in statements", or only when it gets to some longer length?

Reply:
I like breaking statements into newlines tastefully. Although I don't always agree with the results 100%, the clang-format file included with the project is the established code style for the project to keep things consistent.